components/producers/producer.go

```diff
@@ -153,7 +157,7 @@ func getSource() string {
 
 	dat, err := os.ReadFile(sourceMetaPath)
 	if err != nil {
-		log.Println(err)
+		slog.Error(err.Error())
```

Comment:
I don't think you need to pass the `.Error()`, internally the `String()` method will be invoked and the result will be the same

Reply:
i need to pass the error, slog only logs strings :(